pkgs/development/libraries/tinycdb/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, lib, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "tinycdb-${version}";
+  version = "0.78";
+  outputs = [ "out" "dev" "lib" "man" ];
+  separateDebugInfo = true;
+  postPatch = ''
+    substituteInPlace Makefile --replace 'prefix=/usr/local' 'prefix=$(out)'
+  '';
+  buildPhase = ''
+    make staticlib sharedlib cdb-shared
```

Comment:
Anyone know the status of this? https://github.com/NixOS/nixpkgs/issues/28047

Reply:
I was just wondering if you could replace the buildPhase with `buildTargets = [ "staticlib" "sharedlib" "cdb-shared" ];`

Like they use here: https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/browsers/chromium/browser.nix#L8